src/lottery/modules/admin.js

```diff
@@ -0,0 +1,72 @@
+const { useUserCreditAmount } = require("./credit");
+const { transactionModel } = require("./stores/mongo");
+const { recordAction } = require("../../modules/adminResource");
+const { eventEnv } = require("../../../loadenv");
+
+// eventId가 없는 경우 null이 아닌 undefined를 넣어야 합니다.
```

Comment:
```suggestion
/** eventId가 없는 경우 null이 아닌 undefined를 넣어야 합니다. */
```
이렇게 JSDoc으로 바꾸시면 에디터에서 미리보기가 가능합니다!
<img width="623" alt="image" src="https://github.com/sparcs-kaist/taxi-back/assets/46402016/795d686e-926f-4f32-ad12-ddbd148d9af7">